data-service/src/main/java/uk/gov/gchq/palisade/service/data/model/AuditErrorMessage.java

```diff
@@ -188,8 +233,15 @@ public int hashCode() {
     @Generated
     public String toString() {
         return new StringJoiner(", ", AuditErrorMessage.class.getSimpleName() + "[", "]")
-                .add("error=" + error)
-                .add(super.toString())
+                .add("error= " + error)
+                .add("userId= " + userId + "'")
+                .add("resourceId= " + resourceId + "'")
+                .add("context= " + context)
+                .add("serviceName= " + serviceName + "'")
+                .add("timestamp= " + timestamp + "'")
+                .add("serverIP= " + serverIP + "'")
+                .add("serverHostname= " + serverHostname + "'")
+                .add("attributes= " + attributes)
```

Comment:
Why have all the fields been added here? Should we not be using the `super.toString()` method as before?
I would recommend getting rid of the extra space after the `=` as well for consistency across the services.
```suggestion
                .add("error=" + error)
                .add(super.toString())
```

Reply:
done